Scripts/build.sh

```diff
@@ -68,24 +68,24 @@ function xcbuild() {
     if [ $(which xcpretty) ] ; then
         output_command=xcpretty
     fi
+
+    XC_BUILD_ARGS="-version"
     if [[ -n "$XC_DESTINATION" ]]; then
-      xcodebuild \
-        -project "WebDriverAgent.xcodeproj" \
-        -scheme "$XC_TARGET" \
-        -sdk "$XC_SDK" \
-        -destination "$XC_DESTINATION" \
-        $XC_ACTION \
-        $XC_MACROS $EXTRA_XC_ARGS \
-      | $output_command && exit ${PIPESTATUS[0]}
-    else
-      xcodebuild \
-        -project "WebDriverAgent.xcodeproj" \
-        -scheme "$XC_TARGET" \
-        -sdk "$XC_SDK" \
-        $XC_ACTION \
-        $XC_MACROS $EXTRA_XC_ARGS \
-      | $output_command && exit ${PIPESTATUS[0]}
+      XC_BUILD_ARGS+=" -destination ${XC_DESTINATION}"
+    fi
+    if [[ -n "$XC_DERIVED_DATA_PATH" ]]; then
+      XC_BUILD_ARGS+=" -derivedDataPath ${XC_DERIVED_DATA_PATH}"
     fi
+
+    xcodebuild \
+      -project "WebDriverAgent.xcodeproj" \
+      -scheme "$XC_TARGET" \
+      -sdk "$XC_SDK" \
+      "$XC_BUILD_ARGS" \
```

Comment:
should it be quoted?

Reply:
All those quotes (especially if nested in variable) are not resolved properly. To avoid that, I will first create properly formed command as string, then I will execute it using `eval`.

Reply:
As suggested, `eval` was changed to array of arguments.